toolchest_client/query.py

```diff
@@ -25,7 +25,7 @@ class Query():
     """
 
     # Base URLs used by the server API.
-    BASE_URL = "https://api.toolche.st"
+    BASE_URL = os.environ.get("BASE_URL", "https://api.toolche.st")
```

Comment:
Note that this isn't setting a `DEPLOY_ENVIRONMENT` environment variable.

To set, you can create a `.env` file in the root of the project.